packages/idx/src/property/search.component.ts

```diff
@@ -131,11 +131,15 @@ Stratus.Components.IdxPropertySearch = {
          * normal link. Any usable HTML <b>'_target'</b> attribute such as '_self' or '_blank'.
          */
         listLinkTarget: '@',
-        // TODO
+        /**
+         * Type: string
+         * Default: 'advanced'
+         * Options: 'simple' (only a location field), 'basic' (location plus beds/baths/price), 'advanced' (everything)
+         */
         searchType: '@',
         /**
          * Type: string
-         * A link to another dedicated advanced search page (used when this is a module)
+         * A link to another dedicated advanced search page (used when this is a module). NOTE: this should generally be the same as linkListUrl and if not set it will set this to match.
```

Comment:
i believe this was originally for retro gutensite, because theres would be both an advanced full blown filters page vs just the listing page and had been in separate locations